source/jormungandr/jormungandr/instance.py

```diff
@@ -156,6 +158,19 @@ def __init__(
 
         self.zmq_socket_type = zmq_socket_type
 
+        self.configure_equipment_details_providers(equipment_details_providers, name)
+
+    def configure_equipment_details_providers(self, equipment_details_providers, instance_name):
```

Comment:
you don't need to pass `equipment_details_providers, name` as arguments, use `self.*` and I think the code will be little bit more readable with that :)